### ISSUE TITLE
OnActionTagTapped, id and hreaf are null

### ISSUE THREAD

**Issue by KSemenenko** [baskren/Forms9Patch#5]
v. 1.6.6.1
Android

I try use Label, and when I click on the link and call ActionTagTapped then Id and Href - null

**Comment by baskren** [comment]
I'll take a look at this later this week.

**Comment by baskren** [comment]
I retested this functionality in the demo app (see below links) and it works as expected on Android.  If you would like to help with this issue, please send me a link to a github repo that contains a demo project that has the issue you are seeing.

https://github.com/baskren/Forms9PatchDemo/blob/master/Source/Pages.Xaml/XamlCDATA.xaml
https://github.com/baskren/Forms9PatchDemo/blob/master/Source/Pages.Xaml/XamlCDATA.xaml.cs

**Comment by KSemenenko** [comment]
Sorry for the long answer. Here is string that does not work for me.
```
08 Apr 2019 15:45:50:870 <b>HTML</b> <big>really big text</big><a href='http://somesite.gov.ua' target='_blank' title='Official site'>Official site</a>
```

**Comment by baskren** [comment]
Fixed in version 1.6.7.1

**Comment by KSemenenko** [comment]
@baskren  Thanks a lot, you made a wonderful library, thank you!

**Comment by baskren** [comment]
My apologies - a made a big goof and forgot to merge the enhancement into the master branch before releasing.  You'll find the fix in 1.6.7.2.
